modules/administration/pages/ssl-certs.adoc

```diff
@@ -20,5 +20,52 @@ Using a third party CA makes certificates harder to spoof, and will provide addi
 By default, {productname} uses a self-signed certificate.
 If you have certificates signed by a third party CA, you can import them to your {productname} installation.
 
+
+
+== SSL Certificates in {productname}
+// I'm adding this as a separate section for now, to make this easier, but I think we can probably merge these two sections together. --LKB 2020-06-29
+
+Every {productname} Server that uses SSL requires an SSL server certificate.
+Your server certificate is stored using the ``SERVER_CERT`` environment variable.
+
+Every client that uses SSL to register to the {productname} Server checks that it is connecting to the right server by validating against the server certificate.
+This process is called an SSL handshake.
+
+During the SSL handshake, the client will check that the hostname in the server certificate matches.
+The client also needs to check if the server certificate is trusted.
+To do this, it looks for the issuer of the server certificate.
+
+Certificate authorities (CAs) are certificates that are used to sign other certificates.
+The root CA is a special kind of CA that is signed with itself.
+When you look at the details of a root CA, you will see that the subject has the same value as the issuer.
+Your root CA certificate is stored using the ``CA_CERT`` environment variable.
+
+In order for SSL authentication to work correctly, the client must trust the root CA.
+This means that the root CA must be installed on every client.
+
+The default method of SSL authentication is for {productname} to use self-signed certificates.
+In this case, {productname} has generated all the certificates, and the root CA has signed the server certificate directly.
+
+An alternative method is to use an intermediate CA.
+In this case, the intermediate CA signs the root CA.
```

Comment:
I think the oposite. The Root CA signs the Intermediate CA and the Intermediate CA signs other iCA and finally the last one signs the server certificate.

Reply:
Ah yeah, my bad.